src/BlazorUI/Bit.BlazorUI/Components/Surfaces/Accordion/BitAccordion.scss

```
@@ -107,35 +210,74 @@
+
+.bit-acd-cwr {
+    min-height: 0;
     overflow: hidden;
-    padding: 0 spacing(1.5);
-    transition: all $mot-duration-long $mot-easing 0ms;
+}
 
-    // The content region is keyboard-focusable (tabindex=0); same inset outline as the header.
+.bit-acd-con {
+    padding: 0 var(--bit-acd-pad);
+    // Opening: the panel is shown at once and the fade and the padding carry the rest of the way. The
```

Stylelint (17.14.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)